src/Message/PurchaseRequest.php

```diff
@@ -70,11 +70,14 @@ private function sign($params, $api_key) {
 
 	public function sendData($data)
     {
+		// FIXME: You're not actually sending any data here.
```

Comment:
@judgej Ok, thanks. Yeah, the application properly performs the redirect. I was just unsure about the implementation, since @delatbabel added the `FIXME` comment.